examples/cifar10/pt/learners/cifar10_scaffold_learner.py

```diff
@@ -116,7 +117,7 @@ def local_train(self, fl_ctx, train_loader, model_global, abort_signal: Signal,
                     self.save_model(is_best=True)
 
         # Update the SCAFFOLD terms
-        self.scaffold_terms_update(model_global, c_global_para, c_local_para)
+        self.scaffold_helper.terms_update(self.model, model_global, c_global_para, c_local_para, curr_lr)
```

Comment:
It might be good to specify the arg name when calling the function.
So that if we decide to change the arg order, we don't need to change this line of code.

Reply:
changed.